probe-rs/src/architecture/arm/core/exception_handling/armv7m.rs

```diff
@@ -0,0 +1,309 @@
+use crate::{
+    core::{ExceptionInfo, ExceptionInterface},
+    debug::DebugRegisters,
+    memory_mapped_bitfield_register, CoreInterface, Error, MemoryMappedRegister,
+};
+
+use super::cortexm_6_and_7::{self, calling_frame_registers, Xpsr};
```

Comment:
can we name this module something like armv6m_armv7m instead of cortex? Or am I wrong with the entyomology here?

Reply:
Happy to rename ... consider it done :)